### PR TITLE
TN/ITN update

### PR DESCRIPTION
**Pull request by ekmb**
- inference from a raw text file without labels added
- post-processing added to neural TN/ITN and WFST TN
- deleted redundant entries from Ru data

### CODE REVIEW

**Review by yzhang123 — CHANGES_REQUESTED** [review]

File: examples/nlp/duplex_text_normalization/duplex_text_normalization_infer.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+This script contains an example on how to run inference with the DuplexTextNormalizationModel.
+DuplexTextNormalizationModel is essentially a wrapper class around DuplexTaggerModel and DuplexDecoderModel.
+Therefore, two trained NeMo models should be specified to run the joint evaluation
+(one is a trained DuplexTaggerModel and the other is a trained DuplexDecoderModel).
+
+This script can perform inference for 2 settings:
+1. inference from a raw file (no labels required). Each line of the file represents a single example for inference.
+    Specify in inference.from_file and inference.batch_size parameters.
+
+    python duplex_text_normalization_test.py \
```
[the remark this answers is not in the document]
Comment (ekmb) [reply examples/nlp/duplex_text_normalization/duplex_text_normalization_infer.py]: updated

File: examples/nlp/duplex_text_normalization/duplex_text_normalization_infer.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        tagger_pretrained_model=PATH_TO_TRAINED_TAGGER \
+        decoder_pretrained_model=PATH_TO_TRAINED_DECODER \
+        mode={tn,itn,joint} \
+        lang={en,ru,de} \
+        inference.from_file=PATH_TO_RAW_TEXT_FILE
+
+    The predictions will be saved at "_norm" and "_denorm" files.
+
+2. Interactive inference (one query at a time), set inference.interactive to True to enter the interactive mode
+    python duplex_text_normalization_test.py \
```
[the remark this answers is not in the document]
Comment (ekmb) [reply examples/nlp/duplex_text_normalization/duplex_text_normalization_infer.py]: updated

File: examples/nlp/duplex_text_normalization/duplex_text_normalization_infer.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        def _get_predictions(lines: List[str], mode: str, batch_size: int, text_file: str):
+            """ Runs inference on a batch data without labels and saved predictions to a file. """
+            file_name, extension = os.path.splitext(text_file)
+            batch, all_preds = [], []
+            for i, line in enumerate(lines):
+                batch.append(line.strip())
+                if len(batch) == batch_size or i == len(lines) - 1:
+                    outputs = tn_model._infer(
+                        batch,
+                        [constants.DIRECTIONS_TO_MODE[mode]] * len(batch),
```
[the remark this answers is not in the document]
Comment (ekmb) [reply examples/nlp/duplex_text_normalization/duplex_text_normalization_infer.py:91]: I'm breaking it down below, this function accepts only `tn` or `itn` modes, added assert

File: examples/nlp/duplex_text_normalization/duplex_text_normalization_infer.py [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+        while not done:
+            print('Type "STOP" to exit.')
+            test_input = input('Input a test input:')
+            if test_input == "STOP":
+                done = True
+            if not done:
+                outputs = tn_model._infer(
+                    [test_input, test_input],
+                    [
+                        constants.DIRECTIONS_TO_MODE[constants.ITN_MODE],
```
[the remark this answers is not in the document]
Comment (ekmb) [reply examples/nlp/duplex_text_normalization/duplex_text_normalization_infer.py]: done

**Review by yzhang123 — APPROVED** [review]

**Pull request merged**